avocado-plugins-vt.spec

```diff
@@ -7,15 +7,15 @@
 Summary: Avocado Virt Test Plugin
 Name: avocado-plugins-vt
 Version: 51.0
-Release: 0%{?dist}
+Release: 1%{?dist}
 License: GPLv2
 Group: Development/Tools
 URL: http://avocado-framework.readthedocs.org/
 Source0: https://github.com/avocado-framework/%{modulename}/archive/%{commit}/%{modulename}-%{version}-%{shortcommit}.tar.gz
 BuildRequires: python2-devel, python-setuptools
 BuildArch: noarch
 Requires: avocado >= 36.4
-Requires: python, autotest-framework, xz, tcpdump, iproute, iputils, gcc, glibc-headers, python-devel, nc, aexpect, git, python-netaddr, python-netifaces, python-simplejson
+Requires: python, autotest-framework, xz, tcpdump, iproute, iputils, gcc, glibc-headers, python-devel, nc, python-aexpect, git, python-netaddr, python-netifaces, python-simplejson
```

Comment:
Well yes, but I still get `Error: nothing provides avocado >= 36.4 needed by avocado-plugins-vt-51.0-1.fc25.noarch`. Shouldn't that be treated by the `Obsoletes`? Or shouldn't we do something differently?